src/Workspaces/VisualBasic/Portable/FindSymbols/VisualBasicReferenceFinder.vb

```diff
@@ -0,0 +1,46 @@
+﻿Imports System.Composition
+Imports System.Threading
+Imports System.Threading.Tasks
+Imports Microsoft.CodeAnalysis.FindSymbols.Finders
+Imports Microsoft.CodeAnalysis.Host.Mef
+
+Namespace Microsoft.CodeAnalysis.FindSymbols
+    <ExportLanguageService(GetType(ILanguageServiceReferenceFinder), LanguageNames.VisualBasic), [Shared]>
+    Friend Class VisualBasicReferenceFinder
+        Implements ILanguageServiceReferenceFinder
+
+        Public Async Function DetermineCascadedSymbolsAsync(namedType As INamedTypeSymbol, project As Project, cancellationToken As CancellationToken) As Task(Of IEnumerable(Of ISymbol)) Implements ILanguageServiceReferenceFinder.DetermineCascadedSymbolsAsync
+            If namedType.Language = LanguageNames.VisualBasic AndAlso project.Language = LanguageNames.VisualBasic Then
```

Comment:
If either of these aren't true, then something went really wrong, right? Maybe throw an argument exception in that case?